apm-collector/apm-collector-agentstream/src/main/java/org/skywalking/apm/collector/agentstream/worker/service/entry/dao/ServiceEntryH2DAO.java

```diff
@@ -42,10 +41,11 @@
  */
 public class ServiceEntryH2DAO extends H2DAO implements IServiceEntryDAO, IPersistenceDAO<H2SqlEntity, H2SqlEntity> {
     private final Logger logger = LoggerFactory.getLogger(ServiceEntryH2DAO.class);
-    private static final String GET_SERIVCE_ENTRY_SQL = "select * from {0} where {1} = ?";
+    private static final String GET_SERVICE_ENTRY_SQL = "select * from {0} where {1} = ?";
+
     @Override public Data get(String id, DataDefine dataDefine) {
         H2Client client = getClient();
-        String sql = SqlBuilder.buildSql(ServiceEntryTable.TABLE, "id");
+        String sql = SqlBuilder.buildSql(GET_SERVICE_ENTRY_SQL, ServiceEntryTable.TABLE, "id");
```

Comment:
:( I think I have to refactor all these codes in 3.3... since you are using a variable instead of a method to get the table....

Reply:
^_^